app/view_models/nsm/v1/core_cost_summary.rb

```diff
@@ -122,6 +122,10 @@ def letters_calls
       def disbursements
         @disbursements ||= BaseViewModel.build(:disbursement, submission, 'disbursements')
       end
+
+      def additional_fees
+        @additional_fees ||= BaseViewModel.build(:additional_fee, submission)
```

Comment:
Might be worth adding dummy extra `AdditionalFees` to check how they affect areas like this so it doesn't bite us later

Reply:
`additional_fees` now an array of fee objects built via AdditionalFeeSummary object #rows method

Reply:
Sorry I wasn't clear; by that I mean adding test cases to show what having multiple fees looks like.

Reply:
added a dummy view model/dummy additional fee to test this in AdditionalFeeSummary spec